01-ci/task01.py

```diff
@@ -1,21 +1,71 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
+
+"""Compile all tests and examples for all boards.
+
+Usage
+-----
+
+```
+usage: task01.py [-h] [--stop] riot_directory
```

Comment:
Maybe also add an (optional) `RC`/`tag` argument to make sure the script is on the right RC.